README.md

```diff
@@ -1,18 +1,12 @@
 # Functions Lab
 
-## Objectives
-
-1. Practice writing functions
-2. Explain calling functions from within other functions
-2. Practice basics of control flow and `return` statements
-
-## Introduction
+## Problem Statement
 
 Scuber has gained some traction among moms and dads in New York City. It has
```

Comment:
Is Scuber a newly introduced fake business? Can we get a short aside reminding us what Scuber is before talking about what their expectations are?